wiki/Troubleshooting.md

```
@@ -0,0 +1,141 @@
+1. **Check browser console**: Look for JavaScript errors in the browser's developer console.
+2. **Same-origin workaround**: If possible, serve both MagicFrame and the target content from the same origin.
+3. **Verify the onload handler**: Ensure the onload event handler is correctly attached to the iframe.
+
+```javascript
+// Add debugging to check if the onload event fires:
+frame.onload = function() {
+  console.log('iframe loaded');
+  try {
+    document.title = document.getElementById('forwardFrame').contentWindow.document.title;
+    console.log('Title updated to: ' + document.title);
+  } catch (e) {
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)